Specs/DateTime/English/DateExtractor.json

```diff
@@ -1305,11 +1190,90 @@
 ,
 {
   "Input": "on the tenth of September, two thousand",
+  "NotSupported": "javascript,python",
+  "Results": [
+    {
+      "Text": "tenth of September, two thousand",
+      "Type": "date"
+    }
+  ]
+}
+,
+{
+  "Input": "Are you free on 13.5.2015?",
+  "NotSupportedByDesign": "python",
```

Comment:
Why is this not supported in Python if it is in JS? Any specific root cause?

Reply:
Several test cases that were flagged as `NotSupported` in the previous version of JS recognizers are now supported. We decided to temporarily keep them as `NotSupported` until the first version of python date-time recognizer is merged.